### ISSUE TITLE
OpenCV function remap in Tensorflow

### ISSUE THREAD

**Issue by jackieyung** [cysmith/neural-style-tf#34]
Hi, @cysmith 

I found you used the OpenCV function 'remap' to warp the current frame to the last frame. I just want to use  the Tensorflow function for backpropagation. Do you know whether it have the similar function in Tensorflow or not?

Thank you very much.

**Comment by cysmith** [comment]
Hi jackieyung,

No, but I don't think the remap function would be difficult to implement in TF. Are you by any chance trying to implement techniques from [this paper](https://arxiv.org/pdf/1705.02092.pdf)?

Cam

**Comment by jackieyung** [comment]
Hi, cysmith,
Yes, I want to implement the stable fast style transfer from [this paper](https://arxiv.org/pdf/1705.02092.pdf). But sadly I found it used warp method, it need to backpropagation. Tensorflow doesn't have the warp function. 
But I have another question. I have reviewed your video processing code.You just used the 'remap' function to warp current frame to last frame. then used it to calculate loss.The loss also need to backpropagation.My question is when the gradient backpropagating,did it really through the OpenCV 'remap' function? 
Thanks very much.
